api/services/ocean/aws/paths/importASG.yaml

```diff
@@ -1,4 +1,4 @@
-get:
+put:
```

Comment:
@vlad666spot should be 'POST' according to http://help.dev.spotinst.com/spotinst-api/ocean/ocean-cloud-api/ocean-for-aws/import-asg-to-ocean/

Reply:
fixed